nav2_simple_navigator/README.md

```diff
@@ -0,0 +1,17 @@
+# Simple Navigator
+
+The Simple Navigator module implements a **NavigateToPose** task server. As such, its responsibility is to navigate the robot to the specified pose.
```

Comment:
I would add "implements a nav2_tasks::**NavigateToPose**..."

To clarify its implementing an actual defined interface in the nav2_tasks